pillar/heroku/micromasters.sls

```diff
@@ -0,0 +1,212 @@
+      'MAILGUN_FROM_EMAIL': 'no-reply@micromasters-rc-mail.odl.mit.edu',
+      'MAILGUN_URL': 'https://api.mailgun.net/v3/micromasters-rc-mail.odl.mit.edu',
+      'MICROMASTERS_BASE_URL': 'https://micromasters-ci.odl.mit.edu',
+      'MICROMASTERS_CORS_ORIGIN_WHITELIST': "['discussions-ci.odl.mit.edu']",
+      'MICROMASTERS_LOG_LEVEL': 'DEBUG',
+      'MIDDLEWARE_FEATURE_FLAG_QS_PREFIX': '',
+      'OPEN_DISCUSSIONS_API_USERNAME': 'od_mm_ci_api',
+      'OPEN_DISCUSSIONS_BASE_URL': 'https://discussions-ci.odl.mit.edu/',
+      'OPEN_DISCUSSIONS_COOKIE_DOMAIN': 'odl.mit.edu',
+      'OPEN_DISCUSSIONS_COOKIE_NAME': 'discussionsci',
+      'PGBOUNCER_DEFAULT_POOL_SIZE': 100,
+      'PGBOUNCER_MAX_CLIENT_CONN': 1000,
+      'vault_env_path': 'rc-apps'
+      },
+    'rc': {
+      'app_name': 'micromasters-rc',
+      'ALLOWED_HOSTS': '["micromasters-rc.odl.mit.edu"]',
+      'CLOUDFRONT_DIST': 'd3o95baofem9lo',
+      'EDXORG_BASE_URL': 'https://courses.stage.edx.org',
+      'ELASTICSEARCH_INDEX': 'micromasters-rc',
+      'ELASTICSEARCH_SHARD_COUNT': 3,
+      'env_name': 'rc',
+      'FEATURE_EXAMS_CARD_ENABLED': False,
+      'GA_TRACKING_ID': 'UA-5145472-14',
+      'GTM_CONTAINER_ID': 'GTM-NB27R8L',
+      'MAILGUN_FROM_EMAIL': 'no-reply@micromasters-rc-mail.odl.mit.edu',
+      'MAILGUN_URL': 'https://api.mailgun.net/v3/micromasters-rc-mail.odl.mit.edu',
+      'MICROMASTERS_BASE_URL': 'https://micromasters-rc.odl.mit.edu',
+      'MICROMASTERS_CORS_ORIGIN_WHITELIST': "['discussions-rc.odl.mit.edu']",
+      'MICROMASTERS_LOG_LEVEL': 'DEBUG',
+      'MIDDLEWARE_FEATURE_FLAG_QS_PREFIX': 'BGA',
+      'MITXONLINE_BASE_URL': 'https://courses-qa.mitxonline.mit.edu/'
+      'MITXONLINE_CALLBACK_URL': 'https://courses-qa.mitxonline.mit.edu/'
+      'MITXONLINE_URL': 'https://rc.mitxonline.mit.edu/'
+      'OPEN_DISCUSSIONS_API_USERNAME': 'od_mm_rc_api',
+      'OPEN_DISCUSSIONS_BASE_URL': 'https://discussions-rc.odl.mit.edu/',
+      'OPEN_DISCUSSIONS_COOKIE_DOMAIN': 'odl.mit.edu',
+      'OPEN_DISCUSSIONS_COOKIE_NAME': 'discussionsrc',
+      'PGBOUNCER_DEFAULT_POOL_SIZE': 100,
+      'PGBOUNCER_MAX_CLIENT_CONN': 1000,
+      'vault_env_path': 'rc-apps'
+      },
+    'production': {
+      'app_name': 'micromasters-production',
+      'ALLOWED_HOSTS': '["micromasters.mit.edu", "mmfin.mit.edu", "mm.mit.edu"]',
+      'CLOUDFRONT_DIST': 'do5zh7b0lqdye',
+      'EDXORG_BASE_URL': 'https://courses.edx.org',
+      'ELASTICSEARCH_INDEX': 'micromasters',
+      'ELASTICSEARCH_SHARD_COUNT': 2,
+      'env_name': 'production',
+      'FEATURE_EXAMS_CARD_ENABLED': True,
+      'GA_TRACKING_ID': 'UA-5145472-10',
+      'GTM_CONTAINER_ID': 'GTM-NB27R8L',
+      'MAILGUN_FROM_EMAIL': 'no-reply@micromasters.odl.mit.edu',
+      'MAILGUN_URL': 'https://api.mailgun.net/v3/micromasters.odl.mit.edu',
+      'MICROMASTERS_BASE_URL': 'https://micromasters.mit.edu',
+      'MICROMASTERS_CORS_ORIGIN_WHITELIST': "['discussions.odl.mit.edu','odl.mit.edu']",
```

Comment:
Should the second entry be `open.mit.edu`?

Reply:
Currently that's how it's set in the Heroku app, which I don't think makes a lot of sense. Will actually just change it to `open.mit.edu`